### PR TITLE
Add the ASW hack club from Silverflag's vps IP

### PR DESCRIPTION
**Pull request by YetAnotherNotHacking**
Adding `aswhackclub.hackclub.com

## Description

A new hack club is starting at the American School of Warsaw, so we are getting aswhackclub.hackclub.com

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -548,6 +548,10 @@ athena:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+aswhackclub:
```
[the remark this answers is not in the document]
Comment (YetAnotherNotHacking) [reply hackclub.com.yaml:551]: Yah, that would work.

[the remark this answers is not in the document]
Comment (YetAnotherNotHacking) [reply hackclub.com.yaml]: ok I added a second commit if you are able to merge that one, thanks!

it is just asw now.

**Review by alx-alexpark — APPROVED** [review]
lgtm! Just moved up your record because it was slightly out of alphabetical order

**Comment by alx-alexpark** [comment]
Sorry for the late merge here, I didn't see that the check was stuck on awaiting approval because I was using the new beta merge dialogue.

**Pull request merged**